javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/PropagatorsInitializer.java

```diff
@@ -49,7 +52,11 @@ public static void initializePropagators(List<String> propagators) {
       //  https://github.com/open-telemetry/opentelemetry-java/issues/1742
       OpenTelemetry.setGlobalPropagators(
           DefaultContextPropagators.builder()
-              .addTextMapPropagator(HttpTraceContext.getInstance())
+              .addTextMapPropagator(
+                  TraceMultiPropagator.builder()
+                      .addPropagator(HttpTraceContext.getInstance())
+                      .addPropagator(W3CBaggagePropagator.getInstance())
```

Comment:
W3CBaggagePropagator shouldn't be registered as part of `TraceMultiPropagator`, i.e. it should be added directly through `addTextMapPropagator()` ;)

Reply:
ups

Reply:
fixed